fleetshard/pkg/runtime/runtime.go

```diff
@@ -250,28 +250,45 @@ func (r *Runtime) deleteStaleReconcilers(list *private.ManagedCentralList) {
 
 func (r *Runtime) upgradeOperator(list private.ManagedCentralList) error {
 	ctx := context.Background()
+	var desiredOperatorConfigs []operator.OperatorConfig
 	var desiredOperatorImages []string
-	for _, operatorDeployment := range list.RhacsOperators.RHACSOperatorConfigs {
-		glog.Infof("Installing Operator version: %s", operatorDeployment.GitRef)
-		desiredOperatorImages = append(desiredOperatorImages, operatorDeployment.Image)
+
+	if features.UseOperatorsConfigMap.Enabled() {
+		configMapOperators, err := r.operatorManager.ReadOperatorConfigurationFromConfigMap(ctx)
+		if err != nil {
+			glog.Warningf("Failed reading operators configMap: %v", err)
+		}
+
+		if len(configMapOperators) == 0 {
+			glog.Infof("Extracted zero operators from configmap")
+		}
```

Comment:
Imho can be logged unconditionally.

```suggestion

        glog.Infof("Reading operator config map, extracted %d operators from configmap", len(configMapOperators))

```